contracts/src/interfaces/IGateway.sol

```diff
@@ -65,7 +65,11 @@ interface IGateway {
 
     /// @dev Emitted once the funds are locked and an outbound message is successfully queued.
     event TokenSent(
-        address indexed token, address indexed sender, ParaID destinationChain, bytes destinationAddress, uint128 amount
+        address indexed token,
+        address indexed sender,
+        ParaID destinationChain,
+        MultiAddress destinationAddress,
```

Comment:
We may want to re-run e2e before final commit to check if the assets still work.